cv_bridge/include/cv_bridge/cv_bridge.h

```diff
@@ -93,6 +98,9 @@ class CvImage
    */
   void toImageMsg(sensor_msgs::Image& ros_image) const;
 
+  void toCompressedImageMsg(sensor_msgs::CompressedImage& ros_image, const std::string& dst_format = std::string()) const;
```

Comment:
please explain the potential ```dst_format```. Are those actually any standard ? Maybe an OpenCV reference.